src/Neo.CLI/CLI/MainService.cs

```diff
@@ -605,5 +615,32 @@ static string GetExceptionMessage(Exception exception)
 
             return exception.Message;
         }
+
+        static readonly UInt160 nnsHash = UInt160.Parse("0x50ac1c37690cc2cfc594472833cf57505d5f46de");
```

Comment:
Configure the hash inside the config.json?

Reply:
do you know testnet t5 contract hash? There are alot of them. Or you want me to skip testnet?

Reply:
@superboyiii ?

Reply:
> @superboyiii ?

Since the same hash for NNS either on mainnet or testnet, we needn't put it in config.json which may cause further manual issues.